sky/skylet/constants.py

```diff
@@ -415,6 +415,12 @@
 
 # Environment variable that is set to 'true' if metrics are enabled.
 ENV_VAR_SERVER_METRICS_ENABLED = 'SKY_API_SERVER_METRICS_ENABLED'
+
+# Environment variable that is used as the DB connection string for the
+# skypilot server.
+ENV_VAR_DB_CONNECTION_STRING = (
+    f'{SKYPILOT_ENV_VAR_PREFIX}DB_CONNECTION_STRING')
```

Comment:
Should it be URI or URL instead of STRING to be more infromative?
```suggestion
ENV_VAR_DB_CONNECTION_URI = (
    f'{SKYPILOT_ENV_VAR_PREFIX}DB_CONNECTION_URI')
```